content/TrueCommand/TCGettingStarted/Install/_index.md

```diff
@@ -115,23 +110,81 @@ Click the **Discovered Systems** icon and select the TrueNAS system. TrueCommand
+Ensure that you have the "community" package repository enabled:
+* Edit the */etc/apk/repositories* file as root and uncomment the community repository line.
+* Run `apk update` to refresh the list of available packages.
+{{< /hint >}}
+
+Run this command (as root) from a system terminal:   
+`wget https://raw.githubusercontent.com/iXsystems/truecommand-install/main/alpine/setup.sh -O - | sh`
+
+### Void
+
+If you don't already have it, download the VM image [here](https://voidlinux.org/download/).
+
+{{< hint info >}}
+Ensure you have the "wget" utility installed first: `xbps-install -y wget`
+{{< /hint >}}
+
+Run this command (as root) from a system terminal:   
+`wget https://raw.githubusercontent.com/iXsystems/truecommand-install/main/void/setup.sh -O - | bash`
+
+## Windows
+
+If you don't already have it, download the VM image [here](https://www.microsoft.com/en-us/software-download/windows10).
+
+1. On your Windows platform (VM or Bare-Metal) install [Docker for Windows](https://hub.docker.com/editions/community/docker-ce-desktop-windows/).
+2. Open Windows PowerShell (**Start > Windows Power Shell > Windows Power Shell**)
+3. Run the following command inside powershell to start TrueCommand:   
+`docker run --pull=always --restart unless-stopped --detach -v "[hostdirectory]:/data" -p [portnumber]:80 -p [sslportnumber]:443 ixsystems/truecommand`
+
+{{< hint info >}}
+Replace `[hostdirectory]` with a path to where you want TrueCommand to store its local database. Replace `[portnumber]` and `[sslportnumber]` with the ports you wish to expose for TC access.
+{{< /hint >}}
+
+If the command was successful, you should be able to access TrueCommand on http://localhost:80.
+{{< /tab >}}
 {{< tab "Docker (Linux)" >}}
 
 ## Installing the TrueCommand Container
 
 {{< hint warning >}}
-Docker Desktop for Windows uses Hyper-V.
-This interferes with other virtualization applications.
+Docker Desktop for Windows uses Hyper-V, which interferes with other virtualization applications.
 For example, Docker Desktop and VMware Workstation Player cannot simultaneously run.
 {{< /hint >}}
 
 Before fetching the TrueCommand docker image, create a local directory.
-Enter `mkdir {DIRECTORY}`, where *{DIRECTORY}* is the new name.
+Enter `mkdir {DIRECTORY}` replacing `{DIRECTORY}` with the new name.
 
 After creating the new directory, fetch and run the TrueCommand Docker image.
 Open a Command Line Interface (CLI) and enter `docker run \--detach -v "/{HOSTDIR}:/data" -p {PORT}:80 -p
 {SSL}:443 ixsystems/truecommand:latest`.
-*{HOSTDIR}* is a directory on the host machine for Docker container data, *{PORT}* is the TrueCommand web interface port number, and *{SSL}* port number for secure web interface access.
+`{HOSTDIR}` is a directory on the host machine for Docker container data, `{PORT}` is the TrueCommand web interface port number, and `{SSL}` is the port number for secure web interface access.
```

Comment:
Are the names inside the { } brackets variables? if yes, use italics even inside a CLI emphasis for a command line or the reader/user is going to think they need to type exactly what you typed. if HOSTDIR, PORT and SSL are variables for the name of their host directory, port, and SSL port then they are variables, but if these are the actual commands these are correctly typed.